apps/www/components/site-header.tsx

```diff
@@ -5,7 +5,7 @@ import { CommandMenu } from "@/components/command-menu"
 import { Icons } from "@/components/icons"
 import { MainNav } from "@/components/main-nav"
 import { MobileNav } from "@/components/mobile-nav"
-import { ModeToggle } from "@/components/mode-toggle"
+import { ThemeToggle } from "@/components/theme-toggle"
```

Comment:
What are your thoughts on keeping the name to **Mode**Toggle? Here's how I think of it (given that I'll be adding CSS variables soon). 

* Mode is for light and dark mode.
* Theme is for visual like colors, typography...etc.
* A theme can exist in light or dark mode.

Reply:
Sounds good but the name "ModeToggle" may not be clear enough to indicate its purpose. To make it more descriptive, consider renaming it as "ColorModeToggle." or "LightDarkToggle" Maybe you plan to add more predefined themes, such as coffee light, coffee dark, midnight light, and midnight dark so it's should be also considered when naming, and in that case you can keep the name "ThemeToggle" for changing the theme. For customizing colors, consider using a name like "ColorCustomizer" or or "ColorsSettings". But if you only planning dark and light, then "mode" would be fine for it, maybe just rename it to "ColorModeToggle" or smith like this.